daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -207,95 +208,38 @@ private[lf] object Speedy {
 
   private[lf] final case object OffLedger extends LedgerMode
 
-  private[speedy] case class DisclosureTable(
```

Comment:
Disclosure table now only needs to hold contract key mapping information.

Q: should we add this to the `OnLedger` class?